File: INICIANTE/1134/main.py
gasoline = 0
ethanol = 0
diesel = 0
while True:
    fuel = int(input())
    if fuel == 1:
        ethanol += 1
    elif fuel == 2:
        gasoline += 1
    elif fuel == 3:
        diesel += 1
    elif fuel == 4:
        break
print('MUITO OBRIGADO')
print(f'Alcool: {ethanol}')
print(f'Gasolina: {gasoline}')
print(f'Diesel: {diesel}')

File: INICIANTE/1134/main.rb
gasoline = 0
ethanol = 0
diesel = 0

loop do
  fuel = gets.to_i
  if fuel == 1
    ethanol += 1
  elsif fuel == 2
    gasoline += 1
  elsif fuel == 3
    diesel += 1
  elsif fuel == 4
    break
  end
end

puts "MUITO OBRIGADO"
puts "Alcool: #{ethanol}"
puts "Gasolina: #{gasoline}"
puts "Diesel: #{diesel}"
